app/helpers/application_helper.rb

```diff
@@ -430,6 +430,8 @@ def exp_available_tags(model, use_mytags = false)
 
   # Replacing calls to VMDB::Config.new in the views/controllers
   def get_vmdb_config
+    Vmdb::Deprecation.deprecation_warning("ApplicationHelper#get_vmdb_config",
+                                          "Prefer using ::Settings directly.", caller)
```

Comment:
Are deprecation warnings ignored in production?  If so, then this is probably fine as we can still backport to euwe.  If not, we might want to remove this from this PR.